### DISCUSSION TITLE
Easily duplicate station location

### DISCUSSION THREAD

**Discussion by nolith** [magicbug/Cloudlog#1558] [Ideas / Feature Requests]
Hello 👋 
I'm testing cloudlog and I like it. 

I'm doing a lot of POTA/SOTA/outdoor activities and find the current station location management quite hard. 

When I go somewhere I need to create a new location that is ofter identical to another one, just with different SIG/SOTA and maybe the grid. 

What about a quick button to create a new location pre-filled with data from another one? In this way I don't need to set ITU/CQ/eQSL/DXCC/QRZ and all the other field every time I go out portable. 

73 de IU5BON

**Comment by magicbug** [comment]
Yep easy duplication is coming in the coming months with V2 release
